recipe/meta.yaml

```diff
@@ -1,22 +1,24 @@
-{% set year = '2018' %}
-{% set version = year + '0712' %}
-{% set next_year = year|int + 1 %}
+{% set so_number = '152' %}
+{% set epoch = '1' %}
+{% set version_prefix = epoch + '!' + so_number %}
+{% set release_date = '20180712' %}
+{% set version = version_prefix + '.' + release_date %}
 {% set sha256 = "2dfaa34feb7d6dc833ce0f3d0504b31ed409f218d0d8e43c389bb050f0081ff2" %}
 
 package:
   name: x264
   version: {{ version }}
 
 source:
-  fn: x264-snapshot-{{ version }}-2245-stable.tar.bz2
-  url: http://download.videolan.org/pub/videolan/x264/snapshots/x264-snapshot-{{ version }}-2245-stable.tar.bz2
+  fn: x264-snapshot-{{ release_date }}-2245-stable.tar.bz2
+  url: http://download.videolan.org/pub/videolan/x264/snapshots/x264-snapshot-{{ release_date }}-2245-stable.tar.bz2
   sha256: {{ sha256 }}
 
 build:
   number: 0
   skip: true         # [win]
   run_exports:
-    - x264 >={{ version }},<{{ next_year }}0000
+    - x264 {{ version_prefix }}.*
```

Comment:
@sdvillal I am not sure this will work. In semantic versioning, you need to pin the minimum bound to the version that was used during build. This is done in `pin_compatible`. How will this work here?